server/accounts.go

```diff
@@ -2650,17 +2651,16 @@ func (s *Server) updateAccountClaimsWithRefresh(a *Account, ac *jwt.AccountClaim
 		}
 		c.mu.Lock()
 		c.applyAccountLimits()
-		// Check for being revoked here. We use ac one to avoid
-		// the account lock.
-		var nkey string
-		if c.user != nil {
-			nkey = c.user.Nkey
-		}
+		// Check for being revoked here. We use ac one to avoid the account lock.
```

Comment:
I think this comment should be moved down to where we check for ac.Revocations, etc.. because it is confusing here :-)

Reply:
Moved 2 lines down and merged with the comment there